src/app/cmdoptions.cpp

```diff
@@ -332,12 +332,12 @@ QBtCommandLineParameters::QBtCommandLineParameters(const QProcessEnvironment &en
     , skipChecking(SKIP_HASH_CHECK_OPTION.value(env))
     , sequential(SEQUENTIAL_OPTION.value(env))
     , firstLastPiecePriority(FIRST_AND_LAST_OPTION.value(env))
-#ifndef Q_OS_WIN
+#if !defined(Q_OS_WIN) || defined(DISABLE_GUI)
     , showVersion(false)
 #endif
 #ifndef DISABLE_GUI
     , noSplash(NO_SPLASH_OPTION.value(env))
-#else
+#elif defined(DISABLE_GUI) && !defined(Q_OS_WIN)
```

Comment:
Seems `defined(DISABLE_GUI)` is redundant here.

Reply:
it's not, because shouldDaemonize and the whole daemonization is ommited in windows builds

Reply:
if I ommited defined(DISABLE_GUI), it would enable daemonization on unix gui builds and change the previous behavior

Reply:
Should be fixed now, thanks.